nonio-ios/Sources/Features/UserScreen.swift

```diff
@@ -0,0 +1,178 @@
+            content
+            
+            if viewModel.loading {
+                ProgressView()
+            }
+        }
+    }
+    
+    var content: some View {
+        NavigationStack {
+            List {
+                avatarView
+                
+                contentSection
+                    .padding(.top, 20)
+                
+                statsSection
+                    .padding(.top, 24)
+                
+                Button {
+                    settings.currentUser = nil
+                    viewModel.logout()
+                } label: {
+                    Text("Logout")
+                        .font(.title3)
+                        .foregroundColor(.red)
+                }
+                .frame(maxWidth: .infinity)
+                .frame(height: 44)
+                .background(UIColor.secondarySystemBackground.color)
+                .cornerRadius(10)
+                .padding(.horizontal)
+                .padding(.top, 24)
+                .plainListItem()
+            }
+            .toolbar {
+                ToolbarItem(placement: .principal) {
+                    Text("Account")
+                        .font(.headline)
+                        .foregroundColor(.secondary)
+                }
+            }
+            .refreshable {
+                viewModel.onload()
+            }
+            .listStyle(.plain)
+            .navigationBarTitleDisplayMode(.inline)
+            .listRowSeparator(.hidden)
+        }
+        .onLoad {
+            viewModel.onload()
+        }
+
+    }
+    
+    var contentSection: some View {
+        VStack(alignment: .leading) {
+            Text("CONTENT")
+                .foregroundStyle(.secondary)
+                .font(.callout)
+                .frame(height: 40)
+                .multilineTextAlignment(.leading)
+                .padding(.horizontal)
+            
+            VStack {
+                row(
+                    title: "Posts",
+                    value: viewModel.posts,
+                    icon: R.image.postBlue.image,
+                    showIndicator: true
+                )
+                
+                Divider()
+                    .padding(.leading)
+                                        
+                row(
+                    title: "Comments",
+                    value: viewModel.comments,
+                    icon: R.image.commentBlue.image,
+                    showIndicator: true
+                )
+            }
+            .background(UIColor.secondarySystemBackground.color)
+            .cornerRadius(10)
+            .padding(.horizontal)
+        }
+        .plainListItem()
+    }
+    
+    var statsSection: some View {
+        VStack(alignment: .leading) {
+            Text("STATS")
+                .foregroundStyle(.secondary)
+                .font(.callout)
+                .frame(height: 40)
+                .multilineTextAlignment(.leading)
+                .padding(.horizontal)
```

Comment:
Given this is the same style as the CREDENTIALS/CONTENT sections, is there a way we can save this as a class so there's a single thing we change to affect all of these H3 headers?

Reply:
Yes, we can introduce design system to handle all font sizes, colors.